lightning/src/routing/network_graph.rs

```diff
@@ -2110,21 +2030,10 @@ mod tests {
 
 		{
 			// Later announcement which should not be relayed (excess data) prevent us from sharing a node
-			let unsigned_announcement = UnsignedNodeAnnouncement {
-				features: NodeFeatures::known(),
-				timestamp: 1010,
-				node_id: node_id_2,
-				rgb: [0; 3],
-				alias: [0; 32],
-				addresses: Vec::new(),
-				excess_address_data: Vec::new(),
-				excess_data: [1; MAX_EXCESS_BYTES_FOR_RELAY + 1].to_vec(),
-			};
-			let msghash = hash_to_message!(&Sha256dHash::hash(&unsigned_announcement.encode()[..])[..]);
-			let valid_announcement = NodeAnnouncement {
-				signature: secp_ctx.sign(&msghash, node_2_privkey),
-				contents: unsigned_announcement.clone()
-			};
+			let valid_announcement = get_signed_node_announcement(|unsigned_announcement| {
+				unsigned_announcement.timestamp += 10;
+				unsigned_announcement.excess_data = [1; MAX_EXCESS_BYTES_FOR_RELAY + 1].to_vec();
```

Comment:
Hmm, pre-existing but this portion of the test (and possibly other portions) seems to repeat from the `handling_node_announcements` test

Reply:
Yea, it would be nice to clean some of these tests up further, but for now let's leave it as a follow-up? Its already largely-unrelated to this PR.